testing-tools/src/testnet_controllers/strom/strom_internals.rs

```diff
@@ -220,80 +175,52 @@ impl AngstromTestnetNodeInternals {
             let _ = server_handle.stopped().await;
         });
 
-        let testnet_hub = TestnetHub::new(angstrom_addr, state_provider.provider().provider());
-
-        let pool_registry = UniswapAngstromRegistry::new(uniswap_registry, pool_config_store);
-
-        let consensus_handle = ConsensusManager::new(
-            ManagerNetworkDeps::new(
-                strom_network_handle.clone(),
-                state_provider.provider().subscribe_to_canonical_state(),
-                strom_handles.consensus_rx_op
-            ),
-            Signer::new(secret_key),
-            initial_validators,
-            order_storage.clone(),
-            state_provider
-                .provider()
-                .provider()
-                .get_block_number()
-                .await?,
-            pool_registry,
-            uniswap_pools.clone(),
-            state_provider.provider().provider()
-        );
-
-        let _consensus_running = Arc::new(AtomicBool::new(true));
-
-        let _consensus = TestnetConsensusFuture::new(
-            testnet_node_id,
-            consensus_handle,
-            _consensus_running.clone()
-        );
-
-        Ok(Self {
-            rpc_port,
-            state_provider,
-            order_storage,
-            pool_handle,
-            tx_strom_handles,
-            testnet_hub,
-            validator,
-            _consensus,
-            _consensus_running
-        })
+        let testnet_hub =
+            TestnetHub::new(inital_angstrom_state.angstrom_addr, state_provider.rpc_provider());
+
+        // let consensus = if config.is_state_machine() {
+        // let block_number = state_provider
+        //     .provider()
+        //     .provider()
+        //     .get_block_number()
+        //     .await
+        //     .unwrap();
+
+        // let pool_config_store = AngstromPoolConfigStore::default();
+        //let pool_registry = UniswapAngstromRegistry::new(pools.into(),
+        // pool_config_store);
+
+        // let consensus = Some(ConsensusManager::new(
+        //     ManagerNetworkDeps::new(
+        //         strom_network_handle.clone(),
+        //         state_provider.provider().subscribe_to_canonical_state(),
+        //         strom_handles.consensus_rx_op
+        //     ),
+        //     Signer::new(secret_key),
+        //     initial_validators,
+        //     order_storage.clone(),
+        //     block_number + 1,
+        //     pool_registry,
+        //     uni_pools,
+        //     state_provider.provider().provider()
+        // ));
```

Comment:
just delete it. the beauty of git is you can look back instead of cluttering the codebase